### DISCUSSION TITLE
Error sending DTMF

### DISCUSSION THREAD

**Discussion by grelasroberto** [IchthysMaranatha/asterisk-chan-quectel#21] [General]
Hi !
Got it working well in x64 Debian 9, with Asterisk stock (13.14), running as root for voicecalls. However, I'm unable to send DTMF tones from the other end. Asterisk logs for each key pressed in the phone:

[Mar  3 15:09:17] ERROR[18412]: at_response.c:424 log_cmd_response_error: [quectel0] Error sending DTMF
[Mar  3 15:09:18] ERROR[18412]: at_response.c:424 log_cmd_response_error: [quectel0] Error sending DTMF
[Mar  3 15:09:21] ERROR[18412]: at_response.c:424 log_cmd_response_error: [quectel0] Error sending DTMF
[Mar  3 15:09:21] ERROR[18412]: at_response.c:424 log_cmd_response_error: [quectel0] Error sending DTMF

Any clues ??  

thank you so much

**Comment by IchthysMaranatha** [comment]
Will work with inband dtmf

**Comment by IchthysMaranatha** [comment]
Have updated code with quectel/simcom at command to work with modem generated dtmf also
